server/json-logger.cpp

```diff
@@ -12,6 +12,7 @@
 #include "common/fast-backtrace.h"
 #include "common/wrappers/likely.h"
 #include "server/json-logger.h"
+#include "server/php-worker.h"
```

Comment:
why do we need this include?

Reply:
change include to make it more understandable